bundles/tools.vitruv.applications.cbs.testutils/src/tools/vitruv/applications/cbs/testutils/equivalencetest/EquivalenceTestBuilder.xtend

```diff
@@ -0,0 +1,48 @@
+package tools.vitruv.applications.cbs.testutils.equivalencetest
+
+import tools.vitruv.framework.domains.VitruvDomain
+import tools.vitruv.testutils.TestView
+import java.util.function.Consumer
+import org.junit.jupiter.api.DynamicNode
+import tools.vitruv.testutils.TestUserInteraction
+
+/**
+ * Constructs an equivalence test, i.e. a test that executes steps in different domains and checks that if propagated,
+ * the steps lead to the same results.
+ */
```

Comment:
Though this is already merged, it would be helpful to get more documentation. Without knowing the purpose of the equivalence tests, it is hard to understand what is really going on. Therefore it would be nice to document in more detail what "the same results" is (I guess the same models for all domains?) and for steps what a "reference step" means.
From my understanding after reading the code, the equivalence tests are used to test the equivalence of consistency specifications across different domains and equivalence of alternating change sequences within one domain?
@HeikoKlare @jGleitz

Reply:
I found it difficult to write about the concept in code documentation. It is difficult to explain, but not difficult to use, imo. The best documentation would probably be ‘read my master’s thesis’, but that’s not reasonable, of course.

I’ll try to come up with some more docs.